harness/determined/deploy/aws/templates/simple-rds.yaml

```diff
@@ -738,9 +745,6 @@ Resources:
             apt-get update
             apt-get install -y docker-ce docker-ce-cli containerd.io
 
-            curl -fsSL https://www.amazontrust.com/repository/AmazonRootCA1.pem > \
-              /usr/local/determined/etc/db_ssl_root_cert.pem
```

Comment:
just using this cert (like all the others) may've worked. but just using `require` and letting it get resolved through ca certs seems better. when i added tls to postgres like.. 3yrs ago, it didn't work for some reason but now it seems to work fine.